crates/modelardb_server/tests/integration_test.rs

```diff
@@ -168,6 +168,68 @@ fn test_can_get_schema() {
     stop_modelardbd(flight_server);
 }
 
+#[test]
+#[serial]
+fn test_can_list_actions(){
+    let temp_dir = tempfile::tempdir().expect("Could not create a directory.");
+    let flight_server = start_modelardbd(temp_dir.path());
+
+    let runtime = Runtime::new().expect("Unable to initialize runtime.");
+    let mut flight_service_client = create_apache_arrow_flight_service_client(&runtime, HOST, PORT)
+        .expect("Cannot connect to flight service client.");
+
+    let mut actions = Vec::new();
+    runtime.block_on(async {
+        let mut response = flight_service_client
+            .list_actions(Request::new(arrow_flight::Empty {}))
+            .await
+            .expect("Could not retrieve actions.")
+            .into_inner();
+
```

Comment:
The extra newline does not seem necessary and should be removed unless it is required.

Reply:
Fixed.